brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
The "loose" option should now be applied on a plugin-by-plugin basis for those plugins that support it as specified in the [6.0.0 release notes](https://github.com/babel/babel/blob/4c311ec2ab86544a23df866cb74feb25a859f0df/CHANGELOG.md#600).

As an example you can see from the [setup for the test for "loose"](https://github.com/babel/babel/blob/0df276272aff0e590176534c5c5179b839571c7a/packages/babel-plugin-transform-es2015-classes/test/fixtures/loose/options.json) in the class transformer, the options is pushed down further.

If it's important that we activate that flag, then I can change this PR to include each of the 2015 transformations with loose activated for the appropriate plugins:

* `babel-plugin-transform-es2015-for-of`
* `babel-plugin-transform-es2015-template-literals`
* `babel-plugin-transform-s2015-computed-properties`
* `babel-plugin-transform-es2015-classes`

This means that we will not use the `es2015` preset but, rather, list out each of the plugins used for  transpiling to the 2015 standard with the loose option activated.

The new section of the `brunch-config` would look like this:

```javascript
  babel: {
    presets: [],
    plugins: [
      ["babel-plugin-transform-es2015-template-literals", { loose: true }],
      "babel-plugin-transform-es2015-literals",
      "babel-plugin-transform-es2015-function-name",
      "babel-plugin-transform-es2015-arrow-functions",
      "babel-plugin-transform-es2015-block-scoped-functions",
      ["babel-plugin-transform-es2015-classes", { loose: true }],
      "babel-plugin-transform-es2015-object-super",
      "babel-plugin-transform-es2015-shorthand-properties",
      ["babel-plugin-transform-es2015-computed-properties", { loose: true }],
      ["babel-plugin-transform-es2015-for-of", { loose: true }],
      "babel-plugin-transform-es2015-sticky-regex",
      "babel-plugin-transform-es2015-unicode-regex",
      "babel-plugin-check-es2015-constants",
      "babel-plugin-transform-es2015-spread",
      "babel-plugin-transform-es2015-parameters",
      "babel-plugin-transform-es2015-destructuring",
      "babel-plugin-transform-es2015-block-scoping",
      "babel-plugin-transform-es2015-typeof-symbol",
      "babel-plugin-transform-es2015-modules-commonjs",
      ["babel-plugin-transform-regenerator", { async: false, asyncGenerators: false }],
    ],
    ignore: [/web\/static\/vendor/]
  }
  
```